blivet/devicetree.py

```diff
@@ -160,6 +160,12 @@ def _add_device(self, newdev, new=True):
             dev = self.get_device_by_uuid(newdev.uuid, incomplete=True, hidden=True)
             if dev.name == newdev.name:
                 raise DeviceTreeError("Trying to add already existing device.")
+            elif newdev.type == "partition" and dev.type == "partition" and not flags.uevents:
+                # This is a duplicate partition UUID resulting from a cloned disk. Just
+                # pretend it has no UUID. The only negative effect is on event handling,
+                # which is disabled by default.
+                newdev.uuid = None
+                dev.uuid = None
```

Comment:
I don't like the idea of "removing" the UUIDs. We already have two ways how to make the reset "stable" in this situation -- special exception that can be caught and used to ignore one of the disks and the flag. And now we are adding a special case for partitions that just "silently" removes the UUID?. I think the flag should be enough and Anaconda should set it by default.

Reply:
I agree that seems like a sketchy thing to do. I dropped that commit.